jupyter_collaboration/app.py

```diff
@@ -59,5 +64,7 @@ def initialize_handlers(self):
             [
                 (r"/api/yjs/roomid/(.*)", YDocRoomIdHandler),
                 (r"/api/yjs/(.*)", YDocWebSocketHandler),
+                (r"/api/collaboration/room/(.*)", YDocWebSocketHandler),
```

Comment:
Why is there two endpoints with the same handler?

Reply:
The yjs endpoint is deprecated in favor of a more explicit name. Yjs is a framework, not an action or a resource.